docs/mrd_image.md

```diff
@@ -0,0 +1,169 @@
+# MRD Image
+MRD images are stored as a combination of [Image Data](ImageData) and a fixed [Image Header](ImageHeader) of common properties.
+Images can stores as individual 2D images or 3D volumes and may include multiple channels for individual receiver coils.
+
+(ImageHeader)=
+## Image Header
```

Comment:
Feels like we should need this since it is in the model file. Maybe we should add the explanations to the model file instead?